src/utils.ts

```diff
@@ -68,6 +73,28 @@ ajv.addFormat('ethValue', {
   }
 });
 
+const networksIds = Object.keys(networks);
+const mainnetNetworkIds = Object.keys(networks).filter(
+  (id) => !networks[id].testnet
+);
+const testnetNetworkIds = Object.keys(networks).filter(
+  (id) => networks[id].testnet
+);
+
+ajv.addKeyword({
+  keyword: 'snapshotNetwork',
+  validate: function (schema, data) {
+    // @ts-ignore
+    const snapshotEnv = this.snapshotEnv || 'default';
```

Comment:
Why don't we default to `mainnet` ?

Reply:
We now support all networks 🙈 so i thought it is the default case

we can change it later because,
If we make mainnet default, then we need to change the sequencer code before merging this, else demo.snapshot.org will also accept only mainnet networks
Also, UI still displays all networks.

Reply:
So everything works like before, without the 3rd param ?

Reply:
Yep should work